Runtime/Interaction/StaticMesh/HPUIStaticContinuousInteractable.cs

```diff
@@ -0,0 +1,28 @@
+using ubco.ovilab.HPUI.Interaction;
+using UnityEngine;
+
+namespace ubco.ovilab.HPUI.StaticMesh
+{
+    [RequireComponent(typeof(StaticMeshCollidersManager))]
+    public class HPUIStaticContinuousInteractable : HPUIBaseInteractable
```

Comment:
The `ComputeInteractorPostion` needs to be overridden in this class. The interactions won't make sense otherwise.

Reply:
fixed 🫡